web/chromecom.js

```diff
@@ -431,4 +431,44 @@ class MLManager {
   }
 }
 
-export { ExternalServices, initCom, MLManager, Preferences };
+class SignatureStorage {
```

Comment:
This looks quite similar to the implementation in the `genericcom.js` file. (In the latest version of this PR the implementations seem to have diverged slightly.)

To reduce both code duplication and future maintenance burden, could we perhaps instead place the implementation in a new `generic_signature_storage.js` file that both the `web/chromecom.js` and `web/genericcom.js` files import from?